packages/headless/src/controllers/commerce/search-box/headless-search-box-options.ts

```diff
@@ -0,0 +1,23 @@
+import {Schema} from '@coveo/bueno';
+import {
+  searchBoxOptionDefinitions as coreSearchBoxOptionDefinitions,
+  SearchBoxOptions as CoreSearchBoxOptions,
+} from '../../core/search-box/headless-core-search-box-options';
+
+export type SearchBoxOptions = Pick<
+  CoreSearchBoxOptions,
+  'id' | 'highlightOptions' | 'clearFilters'
+>;
+
+type DefaultSearchBoxOptions = Pick<SearchBoxOptions, 'clearFilters'>;
```

Comment:
Is it convention to not use a constant for `clearFilters`?

Reply:
AFAICT, yes